custom_components/tuya_local/devices/edgetheorylabs_coldplunge_heatpump.yaml

```diff
@@ -0,0 +1,77 @@
+name: Edge Theory Cold Plunge Heat Pump
+products:
+  - id: orwtoimcnasji7jw
+    name: Cold Plunge Heat Pump
+primary_entity:
+  entity: climate
+  dps:
+    - id: 121
+      name: hvac_mode
+      type: boolean
+      mapping:
+        - dps_val: false
+          value: "off"
+          icon: "mdi:hvac-off"
```

Comment:
I suggest not to override the icons. HA's defaults are getting better all the time, and this will hide any overall improvements.